### PR TITLE
BNH-82 Letting profile

### PR DESCRIPTION
**Pull request by MaxTheComputerer**
Changes made as a result of testing:
- "Allow pets" radio input is now a required field (you have to choose yes or no)
- Display letting profile on a property's page (looks quite crude, waiting for styling)

### CODE REVIEW

File: web/Views/Property/ViewProperty.cshtml [hunk 1]
```diff
@@ -1,83 +1,112 @@
...
+                </tr>
+                <tr>
+                    <td>Accepts tenants not in EET</td>
+                    <td>@Model.AcceptsNotEET</td>
+                </tr>
+                <tr>
+                    <td>Accepts tenants without a rent guarantor</td>
+                    <td>@Model.AcceptsWithoutGuarantor</td>
+                </tr>
+            </table>
```
Comment (MaxTheComputerer) [inline web/Views/Property/ViewProperty.cshtml:82]: This is the only actual change to this file

File: web/Views/Property/_AddFormStep5.cshtml [hunk 2]
```diff
@@ -25,11 +25,11 @@
...
-        <input type="radio" asp-for="AcceptsPets" value="true" id="PetsTrue" class="form-check-input">
+        <input type="radio" asp-for="AcceptsPets" value="true" id="PetsTrue" class="form-check-input" required>
         <label for="PetsTrue" class="form-check-label">Yes</label>
     </div>
     <div class="form-check">
-        <input type="radio" asp-for="AcceptsPets" value="false" id="PetsFalse" class="form-check-input">
+        <input type="radio" asp-for="AcceptsPets" value="false" id="PetsFalse" class="form-check-input" required>
         <label for="PetsFalse" class="form-check-label">No</label>
     </div>
 </div>
```
Comment (cec200wire) [inline web/Views/Property/_AddFormStep5.cshtml:35]: Maybe a similar check should be applied to the fields 'AcceptsSingleTennant', 'AcceptsCouple' and 'AcceptsFamily' - something which requires at least one option (although possibly more than one option, unlike here) to be selected? It's currently possible to leave all of these unselected, which could lead to landlords accidentally providing information (i.e. by misreading the form as 'Not accept' or something) which says they won't accept any tenants.

Comment (MaxTheComputerer) [reply web/Views/Property/_AddFormStep5.cshtml:35]: Good idea, I'm on it

Comment (BenRamchandani) [reply web/Views/Property/_AddFormStep5.cshtml:35]: That's a good point, we should definitely implement a check.

Might be easier to do after splitting out the pages to have separate view models though if it's difficult to do now..

Comment (MaxTheComputerer) [reply web/Views/Property/_AddFormStep5.cshtml:35]: All done :)

**Review by BenRamchandani — APPROVED** [review]

**Pull request merged**